examples/nidaqmx/analog-input-every-n-samples.py

```diff
@@ -55,49 +55,37 @@ async def _main():
         try:
             client = grpc_nidaqmx.NiDAQmxStub(channel)
 
-            async def raise_if_error(response):
-                """Raise an exception if an error was returned."""
-                if response.status:
-                    response = await client.GetErrorString(
-                        nidaqmx_types.GetErrorStringRequest(error_code=response.status)
+            def check_for_warning(response):
+                """Print to console if the status indicates a warning."""
+                if response.status > 0:
+                    warning_message = client.GetErrorStringResponse(
```

Comment:
I think the method call should be just `GetErrorString` (response is part of the type it returns).

Reply:
Also, is that the right method to be paired with `ErrorMessageRequest` below?

Reply:
Switched to `GetErrorString`.